### PR TITLE
Wip swe 1295 project structure

### PR DESCRIPTION
**Pull request by Anhelina4**

### CODE REVIEW

File: src/app/components/elements/Typography/Text/Text.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types'
+import cn from 'classnames'
+import { createElement } from 'react'
+import styles from './Text.module.css'
+import typographyStyles from '../../Typography/Typography.module.css'
+
+const DEFAULT_TAG = 'p'
```
Comment (AlexFry123) [inline src/app/components/elements/Typography/Text/Text.js:7]: As far as i know, <span> tag is usually used for Text component

File: src/app/components/elements/Typography/Title/Title.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+/**
+ * It returns a React element
+ * @param props - This is the props object that is passed to the component,
+ * has `as` - tag's name.
+ * @returns A React component that renders an HTML element with the tag name specified by the `as`
+ * prop, or `h1` if no `as` prop is provided. The component's class name is set to the tag name.
+ */
+const Title = (props) => {
+  const { as, center, secondary, children, className, ...rest } = props
```
Comment (AlexFry123) [inline src/app/components/elements/Typography/Title/Title.js:17]: Change 'as' property name to 'level' - it's more usual for us as we used 'level' prop name for Title in @qonsoll/react-design

File: src/app/components/layouts/GreetingScreen/GreetingScreen.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  const { title, subtitle, decorationSrc } = props
+  // [COMPUTED PROPERTIES]
+  // get the last character and make it accent color
+  const titleComputed = useMemo(
+    () =>
+      title &&
+      `${title.slice(0, -1)}<span class='text-accent-color'>${
+        title.slice(-1)[0]
+      }</span>`,
+    []
```
Comment (AlexFry123) [inline src/app/components/layouts/GreetingScreen/GreetingScreen.js:33]: Not sure, but looks like you are missing 'title' in dependency array

File: src/app/components/elements/Typography/Title/Title.js (hunk 2 above)
Comment (Anhelina4) [reply src/app/components/elements/Typography/Title/Title.js]: Thanks, i got it. For now I have created folders structure and delete this file. But I will consider your comment in future

File: src/app/components/layouts/GreetingScreen/GreetingScreen.js (hunk 3 above)
Comment (Anhelina4) [reply src/app/components/layouts/GreetingScreen/GreetingScreen.js]: For now I have created folders structure and delete this file. But I will consider your comment in future

File: src/app/components/elements/Typography/Text/Text.js (hunk 1 above)
Comment (Anhelina4) [reply src/app/components/elements/Typography/Text/Text.js]: As I understand this is used for determinig the tag of the element which will be created with React method - createElement.
For now I have created folders structure and delete this file. But I will consider your comment in future, thanks

**Review by GingerBoy14 — APPROVED** [review]

**Pull request merged**